posts/2022-07-07-low-latency-operations-metrics.md

```diff
@@ -0,0 +1,151 @@
+
+To address the different needs of customers, we’re introducing different consistency modes for reading (querying or listing), subscribing, or performing synchronous mutations (Table 1). Clines have the option of providing  their preferred consistency mode when issuing a query, subscription or a synchronous mutation).
+
+**Table 1: Different Consistency Modes**
+
+<table>
+  <tr>
+   <td><strong>Type of Consistency</strong>
+   </td>
+   <td><strong>Dirty Reads</strong>
+   </td>
+   <td><strong>Synchronous E2E Response*</strong>
+   </td>
+   <td><strong>Consistency Guarantees </strong>
+   </td>
+  </tr>
+  <tr>
+   <td><strong>CACHED</strong>
+   </td>
+   <td>Very possible
+   </td>
+   <td>Very fast
+   </td>
+   <td>Transactions are written to a local cache on a given Node and can be out of date up to cache timeout.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>NODE_COMMITTED </strong>
+   </td>
+   <td>Possible
+   </td>
+   <td>Fast
+   </td>
+   <td>Transactions are written to the world state on a given Node but, perhaps, not yet to the ledger.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>NODE_LEDGERED</strong>
+   </td>
+   <td>No
+   </td>
+   <td>Slow
+   </td>
+   <td>Transactions are written to the world state <strong><em>and</em></strong> ledger on a given Node.
+   </td>
+  </tr>
+  <tr>
+   <td><strong>UNI_LEDGERED</strong>
+   </td>
+   <td>No
+   </td>
+   <td>Slowest
+   </td>
+   <td>Transactions are written to the world state <strong><em>and</em></strong> ledger on <strong><em>all</em></strong> Nodes.
+   </td>
+  </tr>
+</table>
+
+*The Synchronous E2E Response is the overall latency as compared to the other consistency modes. Our definition of “slowest” simply means this is the slowest end-to-end latency of all four  consistency modes. 
+
+Prior to this change, `NODE_LEDGERED` was the default mode for reads and synchronous writes. As part of this change, we have made `NODE_COMMITTED` as the default mode if you don’t provide a consistency mode as part of a query, subscription or synchronous mutation. 
+
+Let’s see how you use the new consistency modes with an example: 
+
+**Query Example**
+
+Let’s assume you prefer maximum speed of response over the data consistency. In this case, you will likely use the `CACHED` consistency mode. To select the consistency mode, you add the `readMode` parameter to your query.
+
+```graphql
+query getShapeQuery {
+  get_Shape(id: "0180f1d5-7b0c-c03a-dd22-54eb75b807ef", readMode: CACHED) {
+	color
+	name
+	num_sides
+  
+}
+```
+
+**Synchronous Mutation Example**
+
+Let’s assume you need to ensure that the data is written to at least your local Node before a response is returned. In this case you will likely use the `NODE_LEDGERED` consistency mode. To select the consistency mode, you add the `syncMode` parameter to your mutation. 
```

Comment:
I would change "is written to at least your local node" to "is committed and ledgered on at least your local node"